common/djangoapps/third_party_auth/management/commands/saml.py

```diff
@@ -15,7 +15,9 @@ def add_arguments(self, parser):
         parser.add_argument('--pull', action='store_true', help="Pull updated metadata from external IDPs")
 
     def handle(self, *args, **options):
-        if options['pull']:
+        pull_saml_metadata = options.get('pull', False)
```

Comment:
`options.get` is retrieving the `pull` string. Default should be `None` instead of a `boolean`